### PR TITLE
Simplify handling of orchestrator validator address

### PR DESCRIPTION
**Pull request by skosito**

### CODE REVIEW

File: baseledger/x/bridge/keeper/msg_server_orchestrator_validator_address.go [hunk 1]
```diff
@@ -30,6 +30,14 @@ func (k msgServer) CreateOrchestratorValidatorAddress(goCtx context.Context, msg
 		return nil, sdkerrors.Wrap(sdkerrors.ErrNotFound, "validator not found")
```
Comment (ognjenkurtic) [inline baseledger/x/bridge/keeper/msg_server_orchestrator_validator_address.go:30]: this is not the same check as the one bellow?

File: baseledger/x/bridge/keeper/orchestrator_validator_address.go [hunk 2]
```diff
@@ -1,63 +1,95 @@
...
 	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.OrchestratorValidatorAddressKeyPrefix))
 	store.Delete(types.OrchestratorValidatorAddressKey(
-	    orchestratorAddress,
-    ))
+		orchestratorAddress,
+	))
 }
 
 // GetAllOrchestratorValidatorAddress returns all orchestratorValidatorAddress
 func (k Keeper) GetAllOrchestratorValidatorAddress(ctx sdk.Context) (list []types.OrchestratorValidatorAddress) {
```
Comment (ognjenkurtic) [inline baseledger/x/bridge/keeper/orchestrator_validator_address.go:70]: ..address**es**

Comment (ognjenkurtic) [inline baseledger/x/bridge/keeper/orchestrator_validator_address.go:70]: just for me to understand this returns the mappings orch->val for all nodes?

**Review by ognjenkurtic — APPROVED** [review]

File: baseledger/proto/bridge/genesis.proto [hunk 3]
```diff
@@ -5,15 +5,16 @@ import "gogoproto/gogo.proto";
...
 import "bridge/tx.proto";
+import "bridge/orchestrator_validator_address.proto";
 // this line is used by starport scaffolding # genesis/proto/import
 
 option go_package = "github.com/Baseledger/baseledger/x/bridge/types";
 
 // GenesisState defines the bridge module's genesis state.
 message GenesisState {
   Params params = 1 [(gogoproto.nullable) = false];
+  repeated OrchestratorValidatorAddress orchestratorValidatorAddressList = 5 [(gogoproto.nullable) = false];
```
[the remark this answers is not in the document]
Comment (skosito) [reply baseledger/proto/bridge/genesis.proto:16]: good catch, should be 4

File: baseledger/x/bridge/keeper/msg_server_orchestrator_validator_address.go (hunk 1 above)
Comment (skosito) [reply baseledger/x/bridge/keeper/msg_server_orchestrator_validator_address.go:30]: which one? this is checking if validator exists

File: baseledger/x/bridge/keeper/orchestrator_validator_address.go (hunk 2 above)
Comment (skosito) [reply baseledger/x/bridge/keeper/orchestrator_validator_address.go:70]: yes, generated by starport

**Pull request merged**